libs/application/templates/operating-license/src/forms/draft/subSectionOperationInfo.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  buildMultiField,
+  buildSubSection,
+  buildTextField,
+} from '@island.is/application/core'
+import { m } from '../../lib/messages'
+import { Application } from '@island.is/application/types'
+
+export const subSectionOperationInfo = buildSubSection({
+  id: 'info',
+  title: m.operationInfoTitle,
+  children: [
+    buildMultiField({
+      id: 'info',
+      title: m.operationInfoTitle,
+      description: m.infoSubtitle,
+      children: [
+        buildTextField({
+          id: 'info.operationName',
+          title: m.operationName,
+          width: 'half',
+        }),
+        buildTextField({
+          id: 'info.vskNr',
+          title: m.vskNr,
+          width: 'half',
+          format: '######',
+        }),
+        buildTextField({
+          id: 'info.email',
+          title: m.email,
+          width: 'half',
+          defaultValue: (application: Application) =>
+            (application.externalData.userProfile?.data as {
+              email?: string
+            })?.email,
+        }),
+        buildTextField({
+          id: 'info.phoneNumber',
+          title: m.phoneNumber,
+          width: 'half',
+          variant: 'tel',
+          format: '###-####',
+          defaultValue: (application: Application) =>
+            (application.externalData.userProfile?.data as {
+              mobilePhoneNumber?: string
+            })?.mobilePhoneNumber,
```

Comment:
i think when setting default phone number from user profile, we need to first remove the country code, otherwise the phone number will look something like  f.ex. '354-8417' which is not a valid number

Reply:
i needed to do similar in passport application